README.rst

```diff
@@ -124,24 +124,24 @@ Create database tables::
 Management Commands
 -------------------
 
-``update_topology``
+``update_all``
 ^^^^^^^^^^^^^^^^^^^
```

Comment:
no, don't change this please, or it will break existing deployments who make use of the command.

It's also nicer and more readable to use `./manage.py update_topology`.